dashboard/db/schema.rb

```diff
@@ -947,11 +948,6 @@
   add_index "workshops", ["name"], name: "index_workshops_on_name", using: :btree
   add_index "workshops", ["program_type"], name: "index_workshops_on_program_type", using: :btree
 
-  add_foreign_key "authored_hint_view_requests", "levels"
-  add_foreign_key "authored_hint_view_requests", "scripts"
-  add_foreign_key "authored_hint_view_requests", "users"
-  add_foreign_key "hint_view_requests", "users"
-  add_foreign_key "level_concept_difficulties", "levels"
```

Comment:
Fixed this, then broke it again. I'm going to wait for current tests to finish running, then will push the fix again.